hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeCreateRequest.java

```diff
@@ -132,7 +132,10 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
         }
       }
 
-      UserVolumeInfo volumeList = null;
+      // A replay transaction for CreateVolume can reach here only if the
+      // volume has been deleted in later transactions. Hence, we can
```

Comment:
We can come here without delete request also right. Example CreateVolume T1,  but lastAppliedIndex is at zero, still it it not set to t1(As snapshot has not been taken) and OM restart. Not sure If i am missing something here.

Reply:
You are right. I had updated the check in line 168 but forgot to remove this comment here.